projects/hslayers/src/components/compositions/compositions.service.ts

```diff
@@ -284,7 +284,7 @@ export class HsCompositionsService {
         parsed
       );
       for (let i = 0; i < layers.length; i++) {
-        this.hsMapService.addLayer(layers[i], DuplicateHandling.IgnoreNew);
+        this.hsMapService.addLayer(layers[i], DuplicateHandling.RemoveOriginal);
```

Comment:
If IgnoreNew any layer with created faulty style, bad name, missing features, after fixing and refreshing, will be ignored anyway.

Reply:
Time based and layer configuredd inside app.component.ts probalby were the reason for using IgnoreNew indeed.

Reply:
@raitisbe What should be one about this then?